test/parallel/test-inspector-async-context-brk.js

```diff
@@ -0,0 +1,69 @@
+// Flags: --expose-internals
+'use strict';
+const common = require('../common');
+const { AsyncLocalStorage } = require('async_hooks');
+const als = new AsyncLocalStorage();
+
+function getStore() {
+  return als.getStore();
+}
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+const { Session } = require('inspector');
+const path = require('path');
+const { pathToFileURL } = require('url');
+
+let valueInFunction = 0;
+let valueInBreakpoint = 0;
+
+function debugged() {
+  valueInFunction = getStore();
+  console.log('in code => ', getStore());
+  return 42;
+}
+
+async function test() {
+  const session = new Session();
+
+  session.connect();
+  console.log('Connected');
```

Comment:
I guess logs were just added to illustrate the diff before/after. Clearly they should be removed before merging this.

Reply:
logs removed with latest commit